packages/visualizations/tests/acceptance/table-test.js

```diff
@@ -54,5 +55,21 @@ test('toggle table editing', function (assert) {
   andThen(function () {
     assert.ok(find('.table-header-cell__input').is(':visible'),
       'Table header edit field should be visible');
+
+    assert.notOk(find('.dateTime .table-header-cell__dropdown-trigger').is(':visible'),
+      'Datetime field should not have format dropdown trigger');
+
+    assert.notOk(find('.dimension .table-header-cell__dropdown-trigger').is(':visible'),
+      'Dimension field should not have format dropdown trigger');
+
+    assert.ok(find('.metric .table-header-cell__dropdown-trigger').is(':visible'),
+      'Metric field should have format dropdown trigger');
+
+    clickTrigger();
+    andThen(function () {
+      assert.ok($('.table-header-cell__dropdown-container').is(':visible'),
+        'Table format dropdown should be visible');
+    });
```

Comment:
Do we have an end to end test that updates the format and verifies the new format is reflected in the table?

Reply:
Not yet, since we do not have the table format yet.

Reply:
Can we get the format in then add the test?